app/screens/settings/display_theme/display_theme.tsx

```diff
@@ -26,48 +27,54 @@ type DisplayThemeProps = {
 const DisplayTheme = ({allowedThemeKeys, componentId, currentTeamId, currentUserId}: DisplayThemeProps) => {
     const serverUrl = useServerUrl();
     const theme = useTheme();
-    const initialTheme = useRef(theme);
-    const [newTheme, setNewTheme] = useState<string | undefined>(undefined);
+    const [currentTheme, setCurrentTheme] = useState(theme);
```

Comment:
I think we don't need the `currentTheme` state at all. We only use the variable in the `didUpdate` to check if it has changed, and set it back to `theme`. If I am not mistaken, we don't use it anywhere else.

Reply:
Agreed, changes made.  I was using `currentTheme` in the render, but removed it later.  Didn't think about removing `currentTheme` completely.